### ISSUE TITLE
Tiles colors wrong when COG is not Byte

### ISSUE THREAD

**Issue by Zaitsev** [mojodna/marblecutter-virtual#20]
When COG is float or Int16 tiles becomes wrong-colored(exposition?)

![image](https://user-images.githubusercontent.com/127843/79466267-a54d1580-7ffc-11ea-9a64-3433d0214f59.png)

**Comment by adrienDog** [comment]
Hi, Im having similar issues with 4 band geotiffs, NIR, RGB with uint16 values. did you manage to solve this issue? that would help me a lot thank you

**Comment by Zaitsev** [comment]
Nope. with float bands its like this. converting to uint8 byte helps

**Comment by adrienDog** [comment]
ah okay, thanks a lot for replying so fast :) what do you use to convert to uint8 ? doing a simple ratio makes the image very dark for me
